apiserver/facades/client/application/charmstore.go

```diff
@@ -140,7 +140,11 @@ func AddCharmWithAuthorizationAndRepo(st State, args params.AddCharmWithAuthoriz
 
 	// Validate the charm lxd profile once we've downloaded it.
 	if err := lxdprofile.ValidateCharmLXDProfile(downloadedCharm); err != nil {
-		return errors.Annotate(err, "cannot add charm")
+		if args.Force {
+			logger.Debugf("force flag used to override validation error %v", err)
```

Comment:
This debug msg is duplicated somewhere else and get's printed twice.

Reply:
I'll see what I can do, as we don't pass the error back stating this is a validation error, so we have to debug at a lower level.